docs/current_docs/configuration/cloud.mdx

```diff
@@ -203,6 +203,17 @@ dagger -m github.com/shykes/daggerverse/hello@v0.3.0 call hello
 
 Trigger your CI workflow by pushing a commit or opening a pull request.
 
+### Public traces
+
+Dagger Cloud automatically detects if the traces your pipelines produce are coming
+from CI and a public repository. If so, Dagger Cloud will allow accessing such traces
+without the need to be invited to the organization.
+
+In case the above default behavior needs to be changed, you can do so from the organization
+settings page by toggling the `Trace Repository Visibility` switch under the `Configuration` tab.
+
+![public traces](/img/current_docs/api/public-traces.png)
```

Comment:
We are highlighting sections in images with a red border 6px box. See https://docs.dagger.io/assets/images/manage-volumes-3188bad19829c3dd938d5a2607a0fdea.png for an example. Can you please update the image accordingly? Thanks.

Reply:
![Screenshot 2024-11-06 at 11 29 50 AM](https://github.com/user-attachments/assets/65d20284-1a6e-4346-8de2-80f16c3deb56)


Here is the image to use